### PR TITLE
Dropdown - Disabled test

### PR DESCRIPTION
**Pull request by AUlasen**
Issue: https://github.com/jdi-testing/jdi-light/issues/1119

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java [hunk 1]
```diff
@@ -206,6 +206,8 @@
     public static Dropdown dropdownMenuItems;
     @UI("#dropdown-menu-text-item")
     public static Dropdown dropdownMenuTextItem;
+    @UI("//div[@id='dropdown-menu-disabled-item']/..")
```
Comment (RomE-HK) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java]: Why this locator so complicated?

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DisabledMenuItemTest.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void itemTest() {
+        dropdownMenuDisabledItem.expand();
+        dropdownMenuDisabledItem.list()
+                .get(DISABLED_ITEM_TEXT)
+                .assertThat()
```
Comment (RomE-HK) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DisabledMenuItemTest.java]: Can we use here a JDI assert?

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java (hunk 1 above)
Comment (AUlasen) [reply jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java]: Fixed

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DisabledMenuItemTest.java (hunk 2 above)
Comment (RomE-HK) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DisabledMenuItemTest.java]: Pardon. My fault

**Pull request merged**